src/shadowbox/server/manager_service.ts

```diff
@@ -77,6 +77,10 @@ export function bindService(
     apiServer: restify.Server, apiPrefix: string, service: ShadowsocksManagerService) {
   apiServer.put(`${apiPrefix}/name`, service.renameServer.bind(service));
   apiServer.get(`${apiPrefix}/server`, service.getServer.bind(service));
+  apiServer.put(
+      `${apiPrefix}/server/access-key-data-limit`, service.setAccessKeyDataLimit.bind(service));
+  apiServer.del(
+      `${apiPrefix}/server/access-key-data-limit`, service.removeAccessKeyDataLimit.bind(service));
```

Comment:
I wonder if it's good to model features as a [resource](https://google.aip.dev/121) like we do with access keys (`/server/features/`). That would be similar to `/experimental/` url part we have.

Ideally, I think it would be nice to have something broken down by component like:
* `/server/access-key/keys/:id`
* `/server/access-key/features/:id`
* `/server/access-key/custom-methods`

But that might not be possible to shoehorn to our current scheme.

https://google.aip.dev/

Reply:
In a way we already do this: the access key data limit is a property of the server (resource). I like the idea of formalizing the hierarchy.